packages/flutter_tools/test/general.shard/macos/xcode_test.dart

```diff
@@ -128,210 +116,270 @@ void main() {
     });
 
     group('Xcode', () {
```

Comment:
This section looks like a big change, but it's mostly whitespace due to the the new `group` indentation.  The changes in this file:
1. Remove `xcrun --find xcdevice` commands.
2. Use `FakePlatform` instead of `MockPlatform`.
3. Add the `sysctl` ARM check command where needed.